pipeline/terraform/modules/stack/subsystem_relation_embedder.tf

```diff
@@ -1,7 +1,7 @@
 module "relation_embedder_sub" {
   source = "./relation_embedder"
 
-  namespace        = "relation_embedder_sub"
+  namespace        = "r_embed"
```

Comment:
This namespace has been shortened to allow for maximum name lengths for some resources.